Debug App/Tests/Unit Tests/Analytics/AnalyticsTests.swift

```diff
@@ -1445,14 +1445,7 @@ class AnalyticsTests: XCTestCase {
             try eventsData.write(to: Analytics.Service.filepath)
             
         } catch {
-            primerLogAnalytics(
-                title: "ANALYTICS",
-                message: error.localizedDescription,
-                prefix: "📚",
-                bundle: Bundle.primerFrameworkIdentifier,
-                file: #file, className: "\(Self.self)",
-                function: #function,
-                line: #line)
+            print("📚 ANALYTICS\n📚 \(error.localizedDescription)")
```

Comment:
Is this a special case because it exists only in unit tests? Is it critical for all the tests that this file is created correctly?
I wonder if we should just fail the test with a nice description here...

Definitely a nitpick - aware this requires some refactoring to make this method throw - happy to let it go in this case.